src/components/MapController/MapController.js

```diff
@@ -225,8 +263,57 @@ var MapController = React.createClass({
     return !_.isEqual(nextState, this.state);
   },
 
+  //This function returns JSX for a selector allowing the user to choose
+  //whether a map's colours are scaled logarithmically or linearly.
+  //If a given map cannot be displayed with logscaled colour, returns an
+  //empty string.
```

Comment:
As a user, I found this to be a little confusing. When I went into the settings, I didn't see the selection and was like, "Am I running the right code? Where is the selector?" Would it be better to emit a disabled selector (with the value set to linear) + a tooltip that shows why it is disabled, so that it's more transparent to the user as to why that option is not available?

Reply:
Good point.